### PR TITLE
add support for overriding the default layout

### PR DESCRIPTION
**Pull request by SteveHartzog**
New `overrideLayout` option (if true) will prevent h/w calcs.

### CODE REVIEW

File: js/reveal.js [hunk 1]
```diff
@@ -342,6 +346,7 @@
 		if( typeof query['dependencies'] !== 'undefined' ) delete query['dependencies'];
 
 		// Copy options over to our config object
+		if( typeof(options) === 'object' && options.hasOwnProperty('overrideLayout') && options.overrideLayout ) config.display = '';
```
Comment (hakimel) [inline js/reveal.js]: Can we remove this? Feels a little unexpected that disabling the layout is mutually exclusive with setting the display style style.

**Comment by hakimel** [comment]
Looks like a useful addition, thanks!

I wonder if `disableLayout` is a more appropriate name. The config option disables the default layout logic, it doesn't really override it in itself.

File: js/reveal.js (hunk 1 above)
Comment (SteveHartzog) [reply js/reveal.js]: Yeah, it didn't specifically impact the override... I just didn't want to see it added. :)

**Comment by SteveHartzog** [comment]
Sounds good to me, I'll update my pull to `disableLayout`. and remove the `display` bits. Thanks for reviewing so quickly.

**Comment by hakimel** [comment]
Thanks for updating!

**Pull request merged**